staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -229,6 +233,7 @@ type PodFailurePolicyRule struct {
 	//   counter towards the .backoffLimit is incremented.
 	// Additional values are considered to be added in the future. Clients should
 	// react to an unknown action by skipping the rule.
+	// +required
```

Comment:
Error to be empty https://github.com/kubernetes/kubernetes/blob/e34f01402f2be14e68b03a58c054f1a86bff55b9/pkg/apis/batch/validation/validation.go#L332-L333
Agreed required